buildkite/graphql/pipeline.graphql

```diff
@@ -1,13 +1,16 @@
 fragment PipelineValues on Pipeline {
     id
     allowRebuilds
+    # @genqlient(pointer: true)
     branchConfiguration
     cancelIntermediateBuilds
     cancelIntermediateBuildsBranchFilter
     cluster {
+        # @genqlient(pointer: true)
         id
     }
     defaultBranch
+    # @genqlient(pointer: true)
     defaultTimeoutInMinutes
     maximumTimeoutInMinutes
```

Comment:
This can also be a pointer, I think? `null` is a value option on the `query`.

Reply:
NB: This comment includes changes to other calls in the query and changing to `ValuePointer` as a result in the code.